src/util/versionstore.cpp

```diff
@@ -55,21 +71,17 @@ QString VersionStore::applicationName() {
 QString VersionStore::applicationTitle() {
 #ifdef __APPLE__
```

Comment:
Do we really need the distinction here? I think the application title should always be just `Mixxx`.

Reply:
Yes, I agree. We just need to make sure that this info is inside the about box.

Reply:
I removed `applicationTitle` and use `applicationName` instead. The platform information has been improved and is shown in the About dialog now.